kernel-modules/dockerized/Dockerfile

```diff
@@ -33,7 +31,8 @@ RUN subscription-manager register --username $REDHAT_USERNAME --password $REDHAT
 		git \
 		python3 \
 		kmod && \
-	ln -s /usr/bin/gcc /usr/bin/gcc-8
+	ln -s /usr/bin/gcc /usr/bin/gcc-8 && \
+	subscription-manager unregister
```

Comment:
Out of curiosity, why are we unregistering here but not in `Dockerfile.ubi`?

Reply:
Because we need to copy the rhsm entitlements. `microdnf` doesn't support subscription management (as far as I can tell).